### PR TITLE
TRUNK-6047: Add utility methods to invalidate database caches

### PR DESCRIPTION
**Pull request by docjon09**
## Description of what I changed

Update Context.java
Update ContextTest.java

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-6047

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/context/Context.java [hunk 1]
```diff
@@ -1427,4 +1428,46 @@ public static void setUseSystemClassLoader(boolean useSystemClassLoader) {
...
+
+	/**
+	 * Evicts all entity data from the given region (i.e. for all entities of type).
+	 *
+	 * @param sessionFactory
+	 * @since 2.6.0
+	 */
+	public static void evictAllEntities(SessionFactory sessionFactory, Class<?> entityClass) throw IllegalArgumentException{
+		if(entityClass == null)
+			throws new IllegalArgumentException();
```
Comment (sherrif10) [inline api/src/main/java/org/openmrs/api/context/Context.java:1456]: We can put this on one line

**Comment by sherrif10** [comment]
You also need to follow openmrs pull request convention. Did you forget to create a branch.

**Comment by docjon09** [comment]
> You also need to follow openmrs pull request convention. Did you forget to create a branch.

Do you mean to create a branch on the openMRS repo or the one I forked? Because I am not allowed to create a branch on the this main repo.

**Comment by sherrif10** [comment]
> Do you mean to create a branch on the openMRS repo or the one I forked?

Create the branch on the one you forked and then push

**Pull request closed without merging**